jenkins/jobs/integration_tests.pipeline

```diff
@@ -6,7 +6,7 @@ def CLEAN_TIMEOUT = 600
 def TIMEOUT = 5400
 
 if (env.TESTS_FOR) {
-  if ("${TESTS_FOR}" == "feature_tests") {
+  if ("${TESTS_FOR}" == "feature_tests" || "${TESTS_FOR}" == "feature_tests_upgrade") {
```

Comment:
you could catch both by putting a wildcard in the first match

Reply:
done